app/assets/stylesheets/lessons.scss

```diff
@@ -0,0 +1,723 @@
+// Place all the styles related to the Lessons controller here.
```

Comment:
This file is actually deleted in the redesign, Its been re added because you have the latest commits Erik from master in the PR. You'll have to revert the merge with master commit to fix this.